subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/verification/WriteDependencyVerificationFile.java

```diff
@@ -145,6 +152,7 @@ private void maybeReadExistingFile() throws FileNotFoundException {
     private void writeEntriesSerially() {
         entriesToBeWritten.stream()
             .sorted()
+            .filter(entry -> !isTrustedArtifact(entry.id))
```

Comment:
I wonder if that should not be a fully separate option, to allow controlling trusted artifacts and their presence in the file independently.

Also with these options, the content of a verification file can vary.
I think we should document the minimal file you could create by hand, with your configuration, before populating the file with entries the first time.
Makes sense? (and of course for a later PR)

Reply:
We could split the file in 2, yes. But at this stage I'm not convinced it has a major advantage.

Reply:
Not saying we should split the file in 2, just making sure we document that you can have something like:
```
<?xml version="1.0" encoding="UTF-8"?>
<verification-metadata>
   <configuration>
      <verify-metadata>false</verify-metadata>
      <trusted-artifacts/>
   </configuration>
</verification-metadata>
```
And then  metadata and trusted artifacts would be ignored from the first generation of the file.

Reply:
Right, I don't plan to start documentation until the dogfooding branch passes.